modules/sovrnBidAdapter.js

```diff
@@ -95,7 +95,22 @@ export const spec = {
       });
     }
     return sovrnBidResponses;
-  }
+  },
+
+  getUserSyncs: function(syncOptions, serverResponses) {
+    if (serverResponses && serverResponses.length !== 0 && syncOptions.iframeEnabled) {
+      let iidArr = serverResponses.filter(rsp => rsp.body && rsp.body.ext && rsp.body.ext.iid)
+        .map(rsp => { return rsp.body.ext.iid });
```

Comment:
why are you using map instead of [0].body.ext.iid ?

Reply:
I was thinking of the case with the resulting array is empty and I try to access the zero index element. Wouldn't that cause a console error?

Reply:
good call, and nice solution